tests/inference/models_predictions_tests/test_sam2.py

```diff
@@ -17,11 +17,15 @@
 from inference.core.workflows.core_steps.models.foundation.segment_anything2.v1 import (
     convert_sam2_segmentation_response_to_inference_instances_seg_response,
 )
-from inference.models.sam2 import SegmentAnything2
-from inference.models.sam2.segment_anything2 import (
-    hash_prompt_set,
-    maybe_load_low_res_logits_from_cache,
-)
+try:
+    from inference.models.sam2 import SegmentAnything2
+    from inference.models.sam2.segment_anything2 import (
+        hash_prompt_set,
+        maybe_load_low_res_logits_from_cache,
+    )
+except ModuleNotFoundError:
+    # SAM2 is not installed
```

Comment:
Is this intentional?  seems like imports / anything at module level in those was being tested by CI before that might not now?

Reply:
Yeah, my tests were breaking locally while spinning up even though I wasn't actually running this test.

Reply:
I think we should put it back in / not commit it since we might inadvertently skip test failures that should work on CI

Reply:
actually I guess its fine since this is the test file itself and the tests will fail if the import didn't work

Reply:
Yeh exactly